src/app/cluster-template/component.ts

```diff
@@ -193,7 +193,7 @@ export class ClusterTemplateComponent implements OnInit, OnChanges, OnDestroy {
     const dialogConfig: MatDialogConfig = {
       data: {
         title: 'Delete Cluster Template',
-        message: `Delete <b>${template.name}</b> cluster template permanently? All clusters created using this template will persist.`,
+        message: `Delete <b>${template.name}</b> cluster template permanently? All clusters created using this template will stay.`,
```

Comment:
`All clusters created using this template will stay`

Maybe we can re-word this a bit. Using `stay` here doesn't really make sense.

Reply:
Yea, I agree. Do you have a suggestion?

Reply:
```
```suggestion
        message: `Delete <b>${template.name}</b> cluster template permanently? The clusters created using this template will not be deleted.`,
```